samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/ComposeUiWidgetChildren.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.samples.emojisearch
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateListOf
+import app.cash.redwood.widget.Widget
+
+class ComposeUiWidgetChildren : Widget.Children<@Composable () -> Unit> {
```

Comment:
Promote to redwood-treehouse.

(We could potentially split that into say, `redwood-treehouse-composeui` to avoid imposing a ComposeUI dep on users.)

Reply:
Done.